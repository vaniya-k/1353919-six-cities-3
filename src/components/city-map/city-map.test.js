import React from 'react';
import renderer from 'react-test-renderer';
import CityMap from '../city-map/city-map.jsx';

it(`<CityMap/> should return a Leaflet container`, () => {

  const tree = renderer
    .create(<CityMap placesCoordinates={[{lat: 48.877610000000004, lon: 2.3349911}, {lat: 48.846610000000005, lon: 2.3744911}, {lat: 48.862610000000004, lon: 2.36949911}]} sectionLocationClass={`cities__map`} cityLatLon={{lat: 48.85661, lon: 2.351499}} activePlaceCoordinates={{lat: null, lon: null}}/>)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
