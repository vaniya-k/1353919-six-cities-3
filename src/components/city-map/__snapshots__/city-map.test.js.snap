// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CityMap/> should return a Leaflet container 1`] = `
<section
  className="cities__map map"
>
  <div
    id="mapid"
    style={
      Object {
        "height": "100%",
        "width": "100%",
      }
    }
  />
</section>
`;
